### PR TITLE
Add Brazilian states

### PR DESCRIPTION
**Pull request by larouxn**
Reimplements the `country.json` changes from @jefersonpassos' https://github.com/elementary/website/pull/1818. I originally wanted to simply update that PR to remove the translation changes but I can't revert those changes via the web interface and I can't clone the forked repo branch. Maybe the branch/forked repo has been deleted?
screen shot 2018-01-07 at 12 14 47

### Changes Summary

- Adds Brazilian states

This pull request is ready for review.

### CODE REVIEW

File: data/country.json [hunk 1]
```diff
@@ -39,7 +39,38 @@
     "BM": "Bermuda",
     "BN": "Brunei Darussalam",
     "BO": "Bolivia",
-    "BR": "Brazil",
+    "BR": {
+        "name":"Brasil",
```
Comment (lewisgoddard) [inline data/country.json]: Should this be Brazil?

Comment (larouxn) [reply data/country.json]: I think Brazil is the English spelling, while Brasil is the Brazilian Portuguese spelling.

> Brazil (/brəˈzɪl/ (About this sound listen); Portuguese: Brasil [bɾaˈziw]

[Wikipedia](https://en.wikipedia.org/wiki/Brazil)

Comment (leonardo-lemos) [reply data/country.json]: It should be Brazil to be translated in another moment as Brasil

**Comment by larouxn** [comment]
@leonardo-lemos, I believe this is good to ⛵️ now.

**Pull request merged**